db/migrate/20210712195824_add_url_and_message_to_notifications.rb

```diff
@@ -0,0 +1,6 @@
+class AddUrlAndMessageToNotifications < ActiveRecord::Migration[6.1]
+  def change
+    add_column :notifications, :url, :string
+    add_column :notifications, :message, :text
```

Comment:
You mentioned about these this morning, I think you are right and we will want a null false on both to ensure we don't create any broken notifications with no content or that can't be redirected.